packages/shared/utils/filters.ts

```diff
@@ -38,8 +38,8 @@ export const bypassFilter: EventFilter = (invoke) => {
  *
  * @param ms
  */
-export function debounceFilter(ms: number) {
-  if (ms <= 0)
+export function debounceFilter(ms: MaybeRef<number>) {
+  if (unref(ms) <= 0)
```

Comment:
Then we should check this inside the filter

Reply:
Good catch, will update

Reply:
[]()